docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -1,31 +1,93 @@
-Teleport's plugins use the `access-plugin` user to manage access requests. In order to act as
-the user, the plugin must present the Teleport Auth Service with valid identity files.
-We export the identity files for the user using [`tctl auth sign`](../../setup/reference/cli.mdx#tctl-auth-sign).
+Like all Teleport users, `access-plugin` needs signed credentials in
+order to connect to your Teleport cluster. You will use the `tctl auth sign`
+command to request these credentials for your plugin. 
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+The format of the credentials depends on whether you have set up your network to
+give the plugin direct access to the Teleport Auth Service, or if all Teleport
+clients and services connect to the Teleport Proxy Service instead.
 
 <Tabs>
-  <TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+<TabItem label="Connect to the Proxy Service">
+
+
+The following `tctl auth sign` command impersonates the `access-plugin` user,
+generates signed credentials, and writes an identity file to the local
+directory:
+
+```code
+$ tctl auth sign --user=access-plugin --out=auth.pem
+```
+
+Teleport's Access Request plugins listen for new and updated Access Requests by
+connecting to the Teleport Auth Service's gRPC endpoint over TLS.
+
+The identity file, `auth.pem`, includes both TLS and SSH credentials. Your
+Access Request plugin uses the SSH credentials to connect to the Proxy Service,
+which establishes a reverse tunnel connection to the Auth Service. The plugin
+uses this reverse tunnel, along with your TLS credentials, to connect to the
+Auth Service's gRPC endpoint.
+
+You will refer to this file later when configuring the plugin.
+
+</TabItem>
+<TabItem label="Connect to the Auth Service">
+
+If your network allows your plugin to access the Auth Service directly, e.g.,
+you are running the plugin on the Auth Service host, the plugin uses TLS
+credentials to connect to the Auth Service's gRPC endpoint and listen for new
+and updated Access Requests.
+
+You can generate TLS credentials with the following command:
+
 ```code
 $ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=2190h
```

Comment:
Eventually we should probably make Machine ID the recommended solution for this

Reply:
I've gotta be honest I'm not 100% sure if this is supported, but it sounds to me like what Machine ID should be doing so we can use short-lived certs instead of long-lived, to interact with Teleport. I wouldn't block this PR for it, just the first thought when I see this.

@timothyb89 do you know if tbot would support this today?

Reply:
I believe it should, yup, you'd just give a bot the `access-plugin` role:

```bash
tctl bots add example --roles=access-plugin
```

... then deploy the bot and use the TLS certs it generates (in `/opt/machine-id` or wherever else).

It generates TLS certs, CA certs, and `identityfile`-type certs by default, so it should "just work" for all Teleport API needs without any special configuration.

---

That being said Machine ID is quite a bit more complicated of a solution, there may be some value in keeping `tctl auth sign` instructions around for users who don't want to deploy + maintain another service.

Reply:
Thanks for the information! I've created an issue here: https://github.com/gravitational/teleport/issues/15062